docs/index.md

```diff
@@ -1,53 +1,95 @@
-      community!)
+<div class = "home">
+	<div class = "Xournal" id = "Xournal">
+		<h1>Take all the notes you want with ease, from solving a physics problem to making travel diaries.</h1>
+		<p>Xournal++ is an open-source and cross-platform note-taking software that is fast, flexible, and functional. A modern rewrite and a more feature-rich version of the wonderful Xournal program.</p>
+		<img src = "img/Xournal++ Headline Example.png">
+		<div class = "download-hyperlink upper-button">
+			<a href = "http://127.0.0.1:8000/installation/">Download Xournal++</a>
+		</div>
+	</div>
+	<div class = "feature-list">
+		<div class = "feature feature-right">
+		    <div class = "feature-text">
+		        <h1>Handwrite your notes</h1>
+		        <div class = "feature-description">
+		        	<p class = "bullet-point">|</p><p>Support for pressure-sensitive stylus and drawing tablets (Wacom, Huion, XP-Pen, etc.)</p>
+		        </div>
+		        <div class = "feature-description last-description">
+	        		<p class = "bullet-point">|</p><p>Robust and customizable pen, highlighters, and eraser tool, allowing you to write how you want to write</p>
+	        	</div>
+		    </div>
+		    <img src = "/img/Handwritten.png">
+		</div>
+		<div class = "feature feature-left">
+		    <div class = "feature-text">
+		        <h1>Keep your work organized</h1>
+		        <div class = "feature-description">
+		        	<p class = "bullet-point">|</p><p>Use layers to make complex notes that are still easy to work with</p>
+		        </div>
+		        <div class = "feature-description last-description">
+	        		<p class = "bullet-point">|</p><p>The page previews and PDF bookmarks allow for easy navigation of your notes</p>
+	        	</div>
+		    </div>
+		    <img src = "/img/placeholder-image-2.png">
+		</div>
+		<div class = "feature feature-right">
+		    <div class = "feature-text">
+		        <h1>Record your notes</h1>
+		        <div class = "feature-description last-description">
+	        		<p class = "bullet-point">|</p><p>Record all of your note-taking and the accompanying audio explanation with the built-in audio recorder to share with all of your peers</p>
+	        	</div>
+		    </div>
+		    <img src = "/img/placeholder-image-2.png">
+		</div>
+		<div class = "feature feature-left">
+		    <div class = "feature-text">
+		        <h1>Expedite and facilitate your note-taking</h1>
+		        <div class = "feature-description">
+		        	<p class = "bullet-point">|</p><p>Add images and create various shapes, from circles to splines to axis</p>
+		        </div>
+		        <div class = "feature-description last-description">
+	        		<p class = "bullet-point">|</p><p>Snap objects to rectangular grid or degrees of rotation</p>
+	        	</div>
+		    </div>
+		    <img src = "/img/Create Shape.png">
+		</div>
+		<div class = "feature feature-right">
+		    <div class = "feature-text">
+		        <h1>Extensible to your needs</h1>
```

Comment:
Extensible is quite a mouth full for non-native speakers. Customizable would be somewhat easier to read and would still fit this section quite well.

Reply:
What about changing it to "Suit your needs"?

Reply:
In my opinion, our goal should be to emphasize that the program can be customized _if the user needs to_. "Extensible to your needs" seems off to me, and "Suit your needs" is too generic. Here are some suggestions:
* Heavily customizable
* Optional features for power users
* Easily extended with your own features

Reply:
What about "Do as you wish"?

Reply:
I'd make the main messages more personal, to catch the readers interest:

- Fits all your needs
- Tailored just for you